.circleci/scripts/chrome-install

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+CHROME_VERSION='79.0.3945.117-1'
+CHROME_BINARY="google-chrome-stable_${CHROME_VERSION}_amd64.deb"
+CHROME_BINARY_URL="http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/${CHROME_BINARY}"
```

Comment:
Huh, so the solution to the intermittent Chrome errors was to install it manually? That is interesting.

IIRC Google doesn't leave this URLs up for long, so we may have to frequently bump this. I recall when I was looking for old Chrome versions, I couldn't find any official sources. Perhaps because Google would prefer everybody update.

Reply:
originally we had bumped the chrome version and chromedriver version, causing intermittent failures
@EtDu manually pulled the old chrome version and reverted the chromedriver bump
we should update the chrome/driver pair but its not actually related to this PR

Reply:
Well, it's unfortunate that we'll now have to download and install Chrome each time, slowing down CI further. But we already do that for Firefox, so we're not much worse off now.

Someday we'll get around to creating a custom Dockerfile for CI so we can start with the exact build environment we want.

Reply:
I believe it's a viable solution for the scope of this PR, but we'll definitely want to create a custom image at some point and figure out why the latest chrome is causing intermittent failures.